src/AWS.jl

```diff
@@ -155,7 +155,7 @@ Base.@kwdef mutable struct Request
     request_method::String
 
     headers::AbstractDict{String, String}=LittleDict{String, String}()
-    content::String=""
+    content::Union{String, Array{UInt8}}=""
```

Comment:
```suggestion
    content::Union{String, Vector{UInt8}}=""
```

I don't think it's necessary to support nd arrays here, and keeping the type concrete will help the compiler.